ethcore/light/src/provider.rs

```diff
@@ -174,7 +174,7 @@ impl<T: ProvingBlockChainClient + ?Sized> Provider for T {
 	}
 
 	fn block_receipts(&self, req: request::CompleteReceiptsRequest) -> Option<request::ReceiptsResponse> {
-		BlockChainClient::block_receipts(self, &req.hash)
+		BlockChainClient::encoded_block_receipts(self, &req.hash)
```

Comment:
Doesn't make sense to use `encoded` method here, since we decode it right away, no?